lib/gds_api/test_helpers/publishing_api.rb

```diff
@@ -90,6 +90,39 @@ def publishing_api_isnt_available
         stub_request(:any, /#{PUBLISHING_API_ENDPOINT}\/.*/).to_return(:status => 503)
       end
 
+      def stub_default_publishing_api_put_path
```

Comment:
I think this also needs to be named in a way that makes it clear it's for the path reservations. Maybe something like `stub_default_publishing_api_path_reservation`